src/Nncase.Core/IR/Expr.cs

```diff
@@ -42,8 +54,12 @@ internal Expr(Expr[] operands)
         {
             operand.AddUser(this);
         }
+
+        Metadata = new IRMetadata();
     }
 
+    public IRMetadata Metadata { get; set; }
```

Comment:
Write `public IRMetadata Metadata { get; set; } = new()` instead of initialize in every ctors.